convert.js

```diff
@@ -0,0 +1,408 @@
+  ],
+  majorStar: [
+    "Star",
+    "Star represents hope, a bright future, joy, optimism, guidance, having answers to your questions, being and feeling the connection to the divine, serenity and inspiration. The Star shines so brightly that when it shows up in a reading it tells you that you are being the light in someone’s life.",
+    "Star reversed indicates that any hope or promise offered is going to be false. There is leading astray rather than finding one’s way. There is a feeling of being lost with no way out."
+  ],
+  majorMoon: [
+    "Moon",
+    "Moon represents illusions, intuition, fantasies, fears and anxiety. When the Moon appears things might not be quite as they seem. Your insecurities might be running high or you find yourself on the receiving end of other people’s insecurities.",
+    "Moon reversed indicates primitive forces at play, a long dark night of the soul. Intuition and creativity are blocked, a sexual block is also possible."
+  ],
+  majorSun: [
+    "Sun",
+    "Sun represents success, joy, sunshine, day, warmth and happiness. The Sun shows up when life is sunny and you are enjoying your time with the people you love. Life is simple rather than complicated. Relationships are blossoming and you are feeling loved.",
+    "Sun reversed indicates new beginnings, success and happiness being blocked. Success is blocked due to reaching the wrong conclusions and having bad reasoning and poor logic. Things are frustrating and unclear. There might be some problems with pregnancy or raising young children."
+  ],
+  majorJudgement: [
+    "Judgement",
+    "Judgement represents taking responsibility for your actions and your life, being a good judge of character, seeing the truth and knowing what you want. The Judgement card often shows up when you need to step up and be a leader, speaking the truth and being more assertive.",
+    "Judgement reversed indicates taken on burdens, digging oneself deeper into the old life, being haunted by the past and unable to let go. There is something blocking the querent from renewing themselves."
+  ],
+  majorWorld: [
+    "World",
+    "World is the final Major Arcana card and represents fulfilment and successful completion of a cycle. You know your place in the world, and your life lessons have made you smart and accomplished. The World shows up when the world is ready for you and want what you have to offer.",
+    "World reversed indicates staying at home secluded within the comfort zones, projects and ventures remain incomplete. The querent is unable to finish what they started."
+    ]
+}
+
+var json = Object.keys(old).map(key => {
+  var card = old[key]
+  var assetName = key
+  var name = card[0]
+  var description = card[1]
+  var reverseDescription = card[2]
+  return {
+    assetName: assetName,
+    name: name,
+    description: description,
+    reverseDescription: reverseDescription,
+  }
+})
+
+console.log(JSON.stringify(json, null, 2))
```

Comment:
you're not exporting anything from this file. Are you using it? Also why not store this as JSON in the format you want. Rather than storing one structure in source and then converting it at the end here.